source/common/access_log/access_log_formatter.cc

```diff
@@ -169,6 +171,11 @@ void AccessLogFormatParser::parseCommandHeader(const std::string& token, const s
   } else {
     alternative_header = "";
   }
+  // The main and alternative header should not contain invalid characters {NUL, LR, CF}.
+  if (std::regex_search(main_header, getNewlinePattern()) ||
+      std::regex_search(alternative_header, getNewlinePattern())) {
+    throw EnvoyException("Invalid header configuration. Format string contains newline.");
```

Comment:
Sorry, could you add test coverage for this line?

Reply:
Yes, thanks for catching that!